zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceMigrationMigrateProcessor.java

```diff
@@ -317,7 +317,7 @@ private void tryMigrateElementInstance(
         .ifLeft(
             failure -> {
               throw new ProcessInstanceMigrationPreconditionFailedException(
-                      """
+                  """
```

Comment:
🔧 Since you introduced these changes in a previous commit, you could fix that commit instead of fixing it in a separate commit. This keeps the history clean and your commits focused. Tip: give a try to [git absorb](https://github.com/tummychow/git-absorb), it helps automate this.

Reply:
Just tried, works like a charm! 🚀